jupyterhub/app.py

```diff
@@ -2009,22 +2047,35 @@ async def init_roles(self):
                                 )
                         Class = orm.get_class(bearer)
                         orm_obj = Class.find(db, bname)
```

Comment:
Maybe not for this PR, but add a todo item: if a username appears *only* in a role list, this will return None and raise an unhandled error. role-defined users should probably be added to `init_users`.

Reply:
Done and took tare of the `None` case